client_code/theme.py

```diff
@@ -0,0 +1,306 @@
+            "Primary Overlay 2": "rgba(0, 104, 116, 0.08)",
+            "Primary Overlay 3": "rgba(0, 104, 116, 0.11)",
+        },
+    },
+    "Manarola": {
+        "dark": {
+            "Primary": "#FFB59B",
+            "Primary Container": "#7B2E0E",
+            "On Primary": "#5B1A00",
+            "On Primary Container": "#FFDBCF",
+            "Secondary": "#E7BDB0",
+            "Secondary Container": "#5D4036",
+            "On Secondary": "#442A21",
+            "On Secondary Container": "#FFDBCF",
+            "Tertiary": "#D5C68E",
+            "Tertiary Container": "#50461A",
+            "On Tertiary": "#393005",
+            "On Tertiary Container": "#F2E2A7",
+            "Error": "#FFB4AB",
+            "Background": "#201A18",
+            "Surface": "#201A18",
+            "On Background": "#EDE0DC",
+            "On Surface": "#EDE0DC",
+            "Surface Variant": "#53433E",
+            "On Surface Variant": "#D8C2BB",
+            "Outline": "#A08D86",
+            "On Disabled": "rgba(237, 224, 220, 0.38)",
+            "Disabled Container": "rgba(237, 224, 220, 0.12)",
+            "Light Overlay 1": "rgba(255, 219, 207, 0.08)",
+            "Light Overlay 2": "rgba(255, 219, 207, 0.12)",
+            "Dark Overlay 1": "rgba(255, 219, 207, 0.08)",
+            "Dark Overlay 2": "rgba(255, 219, 207, 0.12)",
+            "Primary Overlay 1": "rgba(255, 181, 155, 0.05)",
+            "Primary Overlay 2": "rgba(255, 181, 155, 0.08)",
+            "Primary Overlay 3": "rgba(255, 181, 155, 0.11)",
+        },
+        "light": {
+            "Primary": "#9A4523",
+            "Primary Container": "#FFDBCF",
+            "On Primary": "#FFFFFF",
+            "On Primary Container": "#380D00",
+            "Secondary": "#77574C",
+            "Secondary Container": "#FFDBCF",
+            "On Secondary": "#FFFFFF",
+            "On Secondary Container": "#2C160D",
+            "Tertiary": "#695E2F",
+            "Tertiary Container": "#F2E2A7",
+            "On Tertiary": "#FFFFFF",
+            "On Tertiary Container": "#221B00",
+            "Error": "#BA1A1A",
+            "Background": "#FFFBFF",
+            "Surface": "#FFFBFF",
+            "On Background": "#201A18",
+            "On Surface": "#201A18",
+            "Surface Variant": "#F5DED6",
+            "On Surface Variant": "#53433E",
+            "Outline": "#85736D",
+            "On Disabled": "rgba(32, 26, 24, 0.38)",
+            "Disabled Container": "rgba(32, 26, 24, 0.12)",
+            "Light Overlay 1": "rgba(255, 255, 255, 0.08)",
+            "Light Overlay 2": "rgba(255, 255, 255, 0.12)",
+            "Dark Overlay 1": "rgba(44, 22, 13, 0.08)",
+            "Dark Overlay 2": "rgba(44, 22, 13, 0.12)",
+            "Primary Overlay 1": "rgba(154, 69, 35, 0.05)",
+            "Primary Overlay 2": "rgba(154, 69, 35, 0.08)",
+            "Primary Overlay 3": "rgba(154, 69, 35, 0.11)",
+        },
+    },
+}
+
+
+class Colours:
```

Comment:
Consider `Colors` over `Colours`

Reply:
Yeah, I've been torn over this one. In the end, I went with the same spelling as is used in the Anvil IDE - the british one.

Reply:
typically anvil does british spelling for docs, and things in the designer, but US spelling for code names

e.g. `anvil.app.theme_colors`
and in the yaml
`parameters.yaml -> color_scheme -> colors`

I don't mind either way
personally I always default to choosing US spelling for code names

Reply:
ah yes. I'd forgotten that little wrinkle! I'll swap the lot to the US version. Even I'm used to typing that in code!